config/database.yml

```diff
@@ -2,6 +2,8 @@ default: &default
   adapter: postgresql
   pool: 5
   timeout: 5000
+  username: <%= ENV['DATABASE_USER'] %>
+  host: <%= ENV['DATABASE_HOST'] %>
```

Comment:
Separate PR seems like the right approach.

I'm happy to crank it out, since it really should have already been done at this point.

Should I yank these lines, once env. variables are set in a .dotenv file?  https://github.com/Ragtagteam/cta-aggregator/pull/21/files#diff-eb672b1ddd3aa7dae5f2577f38daa271R8